odoo/service/server.py

```diff
@@ -162,6 +162,18 @@ def end_headers(self, *a, **kw):
             self.rfile = BytesIO()
             self.wfile = BytesIO()
 
+    @property
+    def path(self):
```

Comment:
It seems this class is used only in the threaded server. What about multi-process?

Reply:
good catch! solved